cmd/tmsh/cmd/root.go

```diff
@@ -19,7 +23,7 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print the version number of tmsh-cli command.",
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("tmsh v0.2.0")
+		fmt.Printf("tmsh %s", version)
```

Comment:
Could you please newline code at the end of the line?

Reply:
Thx a ton. I fixed it.